README.rst

```diff
@@ -37,6 +37,7 @@ numpy            1.9.0
 nibabel          1.4.0dev
 pydicom          0.9.9
 dcmstack         0.7.0dev
+pymongo          2.8
```

Comment:
There is a commit on master (after this branch was created) that eliminates the pymongo/bson dependency.  After I merge your stuff in, I'll bring the whole branch up to date, then we can drop pymongo/bson.